src/components/TotalMarketcapWidget/GetTotalMarketcap.js

```diff
@@ -26,10 +26,33 @@ const getMarketcapQuery = (type, projects) => {
     })
   })
 
-  if (type !== 'list') {
+  const slugsQueryERC20 = graphql(totalMarketcapGQL, {
```

Comment:
In the case when `type` equals to `all` `slugsQueryERC20`variable will be initialized but not used. Better to move it after first return statement (LOC 47)